CRM/Price/BAO/LineItem.php

```diff
@@ -52,11 +52,8 @@ public static function create(&$params) {
         $params['unit_price'] = 0;
       }
     }
-
-    $taxRates = CRM_Core_PseudoConstant::getTaxRates();
     if (isset($params['financial_type_id'], $params['line_total'], $taxRates[$params['financial_type_id']])) {
```

Comment:
this basically means that we won't get into this if because the $taxRates is now undefended I think

Reply:
opps yeah - I need to remove that check - it should be done regardless of whether it is defined

Reply:
updated - thanks!